### DISCUSSION TITLE
Cannot sign up for new account

### DISCUSSION THREAD

**Discussion by dycw** [deta/cloud-docs#512] [General]
I am seeing a `Failed to fetch` error:

![image](https://user-images.githubusercontent.com/10572122/193701544-ffca0e9b-05da-43df-80e1-db087a77e4b3.png)

**Comment by AyushSehrawat** [comment]
Wait for team response...
Till then see what's the issue (ctrl shift i),, try in other browsers / incognito..

**Comment by dycw** [comment]
Sure. Tried your suggestions @AyushSehrawat , and ran into something else:

![Screenshot from 2022-10-04 07-45-18](https://user-images.githubusercontent.com/10572122/193710596-382b1387-76c5-4e5b-966e-fbfa17ae59ac.png)

I never received a registration email (left). Upon hitting reset password, it appears you are trying to send an email to `***@g***`, which is not the email used.

**Reply by AyushSehrawat** [reply]
So you didn't receive the verification mail, but it also attached the wrong mail? 
If that's true, then you can do nothing but contacting @aavshr or @abdelhai

**Reply by AyushSehrawat** [reply]
Try with another email ( probably Gmail ) for now

**Reply by dycw** [reply]
Hi @AyushSehrawat , this is right. I never received anything, and I suspect you are assuming I am using `gmail`.

**Reply by AyushSehrawat** [reply]
Nah, ik you are using icloud, but for the meanwhile you can try on some other email like (gmail)

**Comment by dycw** [comment]
Hi @AyushSehrawat , it turns out only Gmail seems to work.

![image](https://user-images.githubusercontent.com/10572122/193712076-d8dea85d-1222-4dc8-85f8-edc07f94333b.png)

Thanks for your help.

**Reply by AyushSehrawat** [reply]
Hmm.. Well i am sure the team will look into it soon... In the meantime, enjoy deta!

Read the docs at https://docs.deta.sh for queries, or you can join the discord server ( link on https://deta.sh footer)
